src/notation/internal/notationactioncontroller.cpp

```diff
@@ -302,6 +302,7 @@ void NotationActionController::init()
     registerTextAction("text-b", &NotationActionController::toggleBold);
     registerTextAction("text-i", &NotationActionController::toggleItalic);
     registerTextAction("text-u", &NotationActionController::toggleUnderline);
+    registerTextAction("text-u", &NotationActionController::toggleStrike);
```

Comment:
This can't be right, you've assigned both "toggleUnderline" AND "toggleStrike" to the same action!
There's a merge conflict here with my own branch, so I'll just change it to "text-s" for now.

Reply:
Outch!